docs/user-guide/support-level.md

```diff
@@ -0,0 +1,105 @@
+# Support Level
+
+This document provides instructions on how to use the feature support-level API and how to add a new feature and set its support level dynamically.
+
+Support level is a way to determine whether a feature or architecture can be used, and it defines the current support level of that feature. 
+
+The available support-levels are:
+- **supported** - Feature is fully supported.
+- **unsupported** - Feature is not supported for the given parameters.
+- **tech-preview**: Feature is not fully supported, may not be functionally complete, and not suitable for
+  deployment in production. However, these features are provided to the customer as a courtesy and the primary goal
+  is for the feature to gain wider exposure with the goal of full support in the future.
+- **dev-preview**: Feature is considered for possible inclusion into future releases in order to collect feedback about the feature.
+
+The Support Level API has two different endpoints:
+- `GET /v2/support-levels/features` - Get the support level for a given OpenShift version and/or CPU architecture
+- `GET /v2/support-levels/architectures` - Get the supported CPU architectures for a given OpenShift version
+
+Note that the previous API `/v2/feature-support-levels` is now deprecated.
+
+
+## Feature Support Level
+This endpoint returns a list of supported features for a specific OpenShift version. 
+The support level can be evaluated by passing the CPU architecture as a query parameter.
+The result is evaluated dynamically. If a feature isn't supported on the given architecture, passing CPU architecture 
+will change only the support level and not the amount of features.
```

Comment:
It should change the amount of features.  There's a difference between a feature not being supported and not existing.  For example, CNV on ARM - can you install it and it generally works but you can't open bugs on it, or is it impossible to install because there's no ARM image for it?

Reply:
We already decided on that on one of our meeting, we said it is the best to return a list of features and their support level (even if some of them are not supported)

Reply:
That's fine, but unsupported means it exists, but we don't support it.  That's not the same as a feature that simply doesn't exist.

Reply:
In the old code, we also marked it as unsupported, for example, LVM on 4.8 is unsupported, see:
https://github.com/openshift/assisted-service/blob/master/internal/featuresupport/support_levels_list.go#L47 
let's keep the same behavior

Reply:
How does the UI differentiate between:
- a feature that it should block because it doesn't exist for that version/arch
- a feature that exists but the UI should warn about because it's not supported
?

Reply:
cc @ammont82

Reply:
@avishayt I'm trying to understand that with @ammont82, not sure what we currently have logic for that  on our BE

Reply:
According to @ammont82 UI only takes a look on features that is exists.
We currently don't have an indication to a feature that is  not exists for a given parameters (also on the new API)

Reply:
Which of those options is easier for the UI?

Reply:
For UI, the best option is to have the same list of features for every combination of openshift_version & cpu architecture  with "unsupported" value if not exists or if is not supported.

Reply:
"unavailable" is the best I can come up with

Reply:
Thanks

Reply:
I added `unavailable` in the support level description above